bee-message/src/output/feature_block/dust_deposit_return.rs

```diff
@@ -6,27 +6,28 @@ use crate::{
     Error,
 };
 
-use bee_common::packable::{Packable, Read, Write};
+use bee_packable::bounded::BoundedU64;
 
 use core::ops::RangeInclusive;
 
+pub(crate) type DustDepositAmount = BoundedU64<DUST_DEPOSIT_MIN, { u64::MAX }>;
```

Comment:
Use the values from `AMOUNT_RANGE` ?

Reply:
I will, maybe they were introduced after I wrote the code and I didn't catch it during the rebase.